### DISCUSSION TITLE
Badges for Components

### DISCUSSION THREAD

**Discussion by nokome** [codecov/feedback#106] [Ideas]
**What product do you want to improve?**
Web application

**Is your feature request related to a problem? Please describe.**
I want to generate a badge for a Component (like you can for a Flag).

**Describe the solution you'd like**
According to [this](https://community.codecov.com/t/badges-for-flags/270), a badge for a flag can be obtained at:

```
https://codecov.io/gh/<org>/<project>/branch/<branch>/graph/badge.svg?flag=<flag-name>
```

I'd like a badge for a component at 

```
https://codecov.io/gh/<org>/<project>/branch/<branch>/graph/badge.svg?component_id=<component_id>
```

**Describe alternatives you've considered**
None

**Additional context**
None

**Comment by NWilson** [comment]
I agree! That would be ideal for us too.

There is a now an Issue which duplicates this discussion: https://github.com/codecov/feedback/issues/292
